opensrp-goldsmith/src/main/java/org/smartregister/goldsmith/activity/FamilyProfileActivity.java

```diff
@@ -177,4 +177,9 @@ protected void refreshList(Fragment fragment) {
     protected Class<? extends CoreFamilyProfileMenuActivity> getFamilyProfileMenuClass() {
         return null;
     }
+
+    @Override
+    public void onPointerCaptureChanged(boolean hasCapture) {
```

Comment:
I'm just curious. What was the reason for such an implementation?

Reply:
This was erroneously imported